### ISSUE TITLE
Some Old v2 docs still on NUnit.org?

### ISSUE THREAD

**Issue by SeanKilleen** [nunit/nunit.github.io#86]
There appears to still exist a non-canonical NUnit V2 reference on the NUnit.org site:

https://nunit.org/nunitv2/

This also tends to have a lot of the top results for various NUnit searches.

Are these docs published along with this project? I can't seem to locate them in order to submit a PR.

**Comment by CharliePoole** [comment]
The NUnitV2 project was archived in 2015. I temporarily unarchived it to check it out and discovered that we placed a copy of the documentation in a gh-pages branch and published it. At the time, we didn't have the current nunit.org our site was on a host that I provided.

Back when we set up an __organization__ site using GitHub Pages and Jekyll, that NUnitV2 site would have become visible at the nunitv2 subdirectory. That explains the mystery of why we started getting so many results for NUnitV2!

I tried to simply unpublish the site, but it won't let me do that so long as the branch exists. So we need to decide as a group how important this history is. I think it's important but it doesn't need to be easily accessible on the internet. It's fine if you can only see it through GitHub.

I propose to rename the branch to something like `old-gh-pages` and stop the publishing. I'll take it on so long as everyone agrees that's what should be done.

**Comment by SeanKilleen** [comment]
Ah-ha -- nice sleuthing, @CharliePoole!

Are you able to set the branch for github pages to `None` in the repo's settings? 

Example below from `docs`: 

> ![image](https://user-images.githubusercontent.com/2148318/89115178-308ad380-d453-11ea-9dbe-917f61843270.png)

I think the route you proposed makes sense as long as the option above isn't available.

**Comment by CharliePoole** [comment]
Yes, that's the standard way to unpublish. However, it won't work if there's an existing gh-pages branch... an error message appears with a link that takes you .... wait for it.... to the same thing you posted (which doesn't work). thank you github :smiley:

If it were urgent, I would have just done what I proposed, but since it has been a problem for a few years, let's see what others say about the solution, particularly the project leads.

**Comment by ChrisMaddock** [comment]
😂 This mission to preserve links just gets more and more complex, right?

I'd like for us to go down to a single version of the v2 docs - the ones now in the docs repo. _Ideally_ I'd like for the links to the `/nunitv2/` version to redirect to the new authoritative place as well, but I think we need to be realistic about how much time and effort we're prepared to put into this - I think this version might just need to come down.

**Comment by CharliePoole** [comment]
I'm not a big fan of the idea that every link has to be preserved, at least not in this case. The files were copied there in order to __archive__ them, not to make them visible. Unless you knew the secret of `https://nunit.github.io/nunitv2`, you could only get to them through our links on the `nunit.org` documentation page.

Of course, when we moved `nunit.org` to GitHub as an organization-level site, `https://nunit.org/nunitv2` began to work and was subsequently crawled by google. Is it really a problem for us to break those links though?

**Comment by jnm2** [comment]
If I remember right, sites that have broken links can get demoted in Google's search results. It's been a while though so this might not be the case.

**Comment by SeanKilleen** [comment]
I have worked with Google Webmaster Tools before and am able to sort those sorts of things out.

If @rprouse was to grant me access at a domain level (so that I could add TXT records to verify ownership, etc.) I could use the GWT console to sort out the mess of docs and point to `docs.nunit.org` as canonical.  That's entirely up to him though.

**Comment by CharliePoole** [comment]
This is another of those cases where an issue really represents two issues:

1. How to fix the links
2. What to do with the extra copy of legacy docs.

If it weren't the NUnitV2 project, I'd just create a new issue on the project itself for item 2.

However, the NUnitV2 project is archived and can't have issues. Even if it could, it doesn't have a project team or lead so there's nobody to act on any issue.

Since the old NUnitV2 project was supplanted by the current framework and engine/console projects, I'll take the action I proposed on item 2  when I've heard from both project leads. @ChrisMaddock already replied. @rprouse What do you think?

**Comment by CharliePoole** [comment]
@rprouse Ping!

I'm not sure (1) if my renaming proposal still makes sense or (2) if I still have access to carry it out, but either way this seems like a pretty stale issue!

**Comment by rprouse** [comment]
@CharliePoole I think renaming the branch is a good idea.

On a related note, should we be maintaining the v2 documents here, https://docs.nunit.org/?

The styling is all messed up and they look terrible.

**Comment by SeanKilleen** [comment]
@rprouse thank you for reminding me of this. We'd had this working for a bit but something went wrong along the way.

I'll prioritize figuring that out.
